bpf/lib/common.h

```diff
@@ -387,6 +390,9 @@ enum {
 /* Cilium metrics reasons for forwarding packets and other stats.
  * If reason is larger than below then this is a drop reason and
  * value corresponds to -(DROP_*), see above.
+ *
+ * These are shared with pkg/monitor/api/drop.go and api/v1/flow/flow.proto.
```

Comment:
nit: _api/v1/flow/flow.proto_ doesn't expose the "non-drop reasons" (i.e. <130), so it can be removed from this comment.

```suggestion
 * These are shared with pkg/monitor/api/drop.go.
```